package.json

```diff
@@ -48,34 +48,34 @@
     "iterall": "^1.2.2"
   },
   "devDependencies": {
-    "@babel/cli": "7.0.0-beta.53",
-    "@babel/core": "7.0.0-beta.53",
-    "@babel/node": "7.0.0-beta.53",
-    "@babel/plugin-proposal-class-properties": "7.0.0-beta.53",
-    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.53",
-    "@babel/plugin-syntax-async-generators": "7.0.0-beta.53",
-    "@babel/plugin-transform-classes": "7.0.0-beta.53",
-    "@babel/plugin-transform-destructuring": "7.0.0-beta.53",
-    "@babel/plugin-transform-flow-strip-types": "7.0.0-beta.53",
-    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.53",
-    "@babel/plugin-transform-spread": "7.0.0-beta.53",
-    "@babel/polyfill": "7.0.0-beta.53",
-    "@babel/preset-env": "7.0.0-beta.53",
-    "@babel/register": "7.0.0-beta.53",
+    "@babel/cli": "7.0.0-beta.55",
+    "@babel/core": "7.0.0-beta.55",
+    "@babel/node": "7.0.0-beta.55",
+    "@babel/plugin-proposal-class-properties": "7.0.0-beta.55",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.55",
+    "@babel/plugin-syntax-async-generators": "7.0.0-beta.55",
+    "@babel/plugin-transform-classes": "7.0.0-beta.55",
+    "@babel/plugin-transform-destructuring": "7.0.0-beta.55",
+    "@babel/plugin-transform-flow-strip-types": "7.0.0-beta.55",
+    "@babel/plugin-transform-modules-commonjs": "7.0.0-beta.55",
+    "@babel/plugin-transform-spread": "7.0.0-beta.55",
+    "@babel/polyfill": "7.0.0-beta.55",
+    "@babel/preset-env": "7.0.0-beta.55",
+    "@babel/register": "7.0.0-beta.55",
     "babel-eslint": "8.2.6",
     "babel-plugin-syntax-async-functions": "6.13.0",
     "beautify-benchmark": "0.2.4",
     "benchmark": "2.1.4",
     "chai": "4.1.2",
     "coveralls": "3.0.2",
-    "eslint": "5.1.0",
+    "eslint": "5.2.0",
     "eslint-plugin-babel": "5.1.0",
     "eslint-plugin-flowtype": "2.50.0",
     "eslint-plugin-prettier": "2.6.2",
-    "flow-bin": "0.76.0",
+    "flow-bin": "0.77.0",
     "mocha": "5.2.0",
     "nyc": "12.0.2",
-    "prettier": "1.13.7",
-    "sane": "2.5.2"
+    "prettier": "1.14.0",
+    "sane": "3.0.0"
```

Comment:
How valuable is the `sane` upgrade? I know we would be stuck on `2.5` for a very long time if we don't upgrade it now, but the release nodes for `3.0` have nothing in them, which makes me think they bumped to `3.0` just so they could stop supporting node v9.

Reply:
@mjmahone It's only a dev dependency and we use it only for `yarn watch` which is broken anyway. 
So I can easily revert it but as I wrote in #1445 I don't see any problem with dropping Node 9